tests/integration/ovs_test.py

```diff
@@ -204,12 +193,18 @@ def test_ovs_remove_port(bridge_with_ports):
 
 @contextmanager
 def disable_nm_ovs_plugin():
-    cmdlib.exec_cmd(DNF_REMOVE_NM_OVS_CMD)
-    cmdlib.exec_cmd(SYSTEMCTL_RESTART_NM_CMD)
+    tmp_path = "/tmp/libnm-device-plugin-ovs.so"
```

Comment:
I would prefer not to use a hard-coded path in `/tmp` but to either create a proper temporary directory with `tempfile.mkdtemp` or to rename the file inside the actual directory so it will not move across file systems. You can then use `os.rename` to move the file.

Reply:
I cannot use the os.rename(). If I use it then I am getting an error because of a symlink. I will reproduce it and paste here the error. I am fine with creating a temporary directory.

Reply:
You cannot use `os.rename` to move the file from /usr/lib64/NetworkManager/... to /tmp but you could use it to rename the plugin from `libnm-device-plugin-team.so` to `libnm-device-plugin-team.nmstate_disabled` or similar.

Reply:
I did the @EdDev approach. What do you think? Thanks!